### PR TITLE
polish trace

### PR DESCRIPTION
**Pull request by tydhot**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by ztelur** [comment]
because some legacy issue of code, should the change of pkg/client/http/http.go also apply to filter/http/httpproxy/routerfilter.go too ?

**Comment by tydhot** [comment]
> because some legacy issue of code, should the change of pkg/client/http/http.go also apply to filter/http/httpproxy/routerfilter.go too ?

In this PR, I just optimize the trace code in the existing project. Idk why filter/http/httpproxy/routerfilter.go  didn't have any trace code. Maybe should it add in this file?

**Review by AlexStocks — APPROVED** [review]

**Review by ztelur — APPROVED** [review]

File: pkg/client/http/http.go [hunk 1]
```diff
@@ -24,14 +24,12 @@ import (
 	"sync"
 	"time"
```
Comment (xiaoliu10) [inline pkg/client/http/http.go:25]: 拆分import 块 分为 语言的, 本项目的, 外部引用的

**Review by mark4z — APPROVED** [review]

File: pkg/client/http/http.go (hunk 1 above)
Comment (mark4z) [reply pkg/client/http/http.go:25]: There are a tool to fix it,see "tools" in dubbo-go profile

Comment (tydhot) [reply pkg/client/http/http.go:25]: got it

**Pull request merged**